pyproject.toml

```diff
@@ -41,6 +41,14 @@ dependencies = [
 ]
 
 [project.optional-dependencies]
+alphanet = [
+    "alphanet @ git+https://github.com/zmyybc/AlphaNet.git",
+    "numpy<2.0",
```

Comment:
Does it not work with numpy 2+? I didn't see anything in their requirements etc. about numpy explicitly?